src/core/renderers/webgl/WebGLRenderer.js

```diff
@@ -376,6 +376,23 @@ export default class WebGLRenderer extends SystemRenderer
     }
 
     /**
+     * Erases the render texture and fills the drawing area with a colour
+     *
+     * @param {PIXI.RenderTexture} renderTexture - The render texture to clear
+     * @param {number} [clearColor] - The colour
+     */
+    clearRenderTexture(renderTexture, clearColor)
+    {
+        const baseTexture = renderTexture.baseTexture;
+        const renderTarget = baseTexture._glRenderTargets[this.CONTEXT_UID];
+
+        if (renderTarget)
+        {
+            renderTarget.clear(clearColor);
+        }
+    }
```

Comment:
Could you `return this;` to be consistent with the other methods in the Renderer? Also, don't forget the JSDoc for return (can be copied from the other methods).